project_templates/test_report/TESTTR-0/.github/workflows/docgen_from_Jira.yaml

```
@@ -37,9 +37,10 @@ jobs:
  
       - name: Create commit
         run: |
-          git config user.name github-actions
-          git config user.email github-actions@github.com
+          git config user.name $GITHUB_ACTOR
+          git config user.email $GITHUB_ACTOR@users.noreply.github.com
           git commit -m "Update from JIRA." TESTTR-0.tex 
```

```suggestion
          git add .
          git commit -m "Update from JIRA." TESTTR-0.tex 
```

I think the `git add` is needed since docsteady seems to be adding untracked files according to https://github.com/lsst-dm/DMTR-301/runs/2239397745?check_suite_focus=true

---

And then drop the filename from the `git commit` command